plugins/modules/system/runit.py

```diff
@@ -239,7 +239,6 @@ def main():
             name=dict(type='str', required=True),
             state=dict(type='str', choices=['killed', 'once', 'reloaded', 'restarted', 'started', 'stopped']),
             enabled=dict(type='bool'),
-            dist=dict(type='str', default='runit'),
```

Comment:
This is a breaking change. Instead, this option should be deprecated (and not documented) for removal in 3.0.0, since it doesn't seem to be used at all. But also that's something that should be done in a separate PR.